docs/REST API/rest-api.md

```diff
@@ -2009,7 +2012,10 @@ If `after` is being used with `created_at` or `updated_at`, the table must be sp
         "encryption_key_available": false,
         "enrollment_status": null,
         "name": "",
-        "server_url": null
+        "server_url": null,
+        "locked": false,
+        "unlock_pin": "",
```

Comment:
```suggestion
```

Reply:
This is so we know when a user has viewed the pin, moving it to a separate endpoint